StyleGuide.md

```diff
@@ -5,98 +5,219 @@ The following guidance is provided to benefit CockroachDB docs authors and revie
+  **Prefer:** CockroachDB does not support column-level privileges.
+
+### Latinisms
+
+For readability, avoid Latinisms.
+
+- **Avoid:** Select a deployment option, e.g., {{ site.data.products.standard }} or {{ site.data.products.advanced }}.  
+  **Prefer:** Select a deployment option such as {{ site.data.products.standard }} or {{ site.data.products.advanced }}.
+
+## Word usage guidelines
 
-- Avoid using forward-looking language when writing about supported syntax and behavior:
-    - Do not suggest that a feature may or may not be added in a future release.
-    - Do not use the words "yet" and "currently" when writing about a feature that we do or do not support.
-    - Do not reference the internal product roadmap.
+This section logs decisions about dictionary words to use and avoid. Add specific guidelines we decide as a team to this section. Refer also to [Technical terminology](#technical-terminology) for technical terms that may not be in a dictionary.
 
-## Inclusive language
+### CockroachDB products
 
-We want our education materials to be inclusive and written with diversity in mind. This section provides general guidelines, best practices, and examples when writing docs or training materials. This is a work in progress and we welcome any feedback and additions.
+- Use the following combinations of `CockroachDB` and Liquid variables to display product names on docs pages. Liquid variables cannot be used in Markdown front-matter, file names, or non-Markdown files. In those cases, write the product names manually.
+
+  - **CockroachDB Cloud** : `CockroachDB {{ site.data.products.cloud }}`
+  - **CockroachDB Basic** : `CockroachDB {{ site.data.products.basic }}`
+  - **CockroachDB Standard** : `CockroachDB {{ site.data.products.standard }}`
+  - **CockroachDB Advanced** : `CockroachDB {{ site.data.products.advanced }}`
+  - **CockroachDB self-hosted cluster** : `CockroachDB {{ site.data.products.core }} cluster`
+
+- Use the full product name on its first occurrence on a page. You may subsequently shorten product names to "Basic", "Advanced", or "Cloud", unless this would introduce ambiguity (for example, on long pages where product names might be separated by multiple paragraphs).
+
+- Do not shorten "CockroachDB" to "CRDB".
+
+- Use "`cockroach`" to refer to the CockroachDB binary.
+
+  - **Avoid:** Cockroach process  
+    **Prefer:** `cockroach` process
+
+### CockroachDB versions
+
+Refer to CockroachDB versions as `vxx.x.x` (for example, `v21.1.8`). Do not use `version xx.x.x`.
+
+For version ranges, use `to` between numbers (for example, `v22.1.0 to v22.1.4`). Do not use a dash. Refer to [Punctuation rules](#punctuation-rules).
+
+Refer to "earlier" and "later" versions, rather than "lower" and "higher" versions. For example, `CockroachDB v25.3 and earlier`.
+
+### Third-party products
+
+In general, align third-party branding with that brand's usage. Do not use the shortened versions of product names.
```

Comment:
i have received feedback in the past to avoid linking to 3rd party products' docs unless absolutely necessary for clarity.  IDK if that is appropriate for inclusion in this section, or if we would like to discuss it more separately. I don't have a strong opinion _except_ that such external links often break because humans break their websites constantly

Reply:
I have the possibly total opposite opinion, which is that I don't want us to be responsible for documenting third party product flows and would rather link out to their documentation wherever possible for those. Easier to identify and fix a broken link than correct text on our site that describes an outdated third-party procedure.

Reply:
Linking out to external products is absolutely necessary for things like migration docs. I think I can add some follow-up guidance on external links that is consistent with both of your comments; stay tuned.